export.yml

```diff
@@ -0,0 +1,48 @@
+# This example workflow demonstrates how to create a parameterized Google Sheets Export job.
+#
+# Workflows in the Civis Platform are written in YAML and a workflow DSL
+# (domain specific language) called Mistral.
+#
+# See this website, https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html,
+# for an introduction to YAML.
+#
+# See the Mistral documentation, https://docs.openstack.org/mistral/pike/user/dsl_v2.html,
+# for a description of the Mistral DSL.
+
+version: '2.0' # you always need this key to specify version 2 of the mistral DSL
+
+google_export_workflow: # the name of the workflow can be anything, but Platform ignores it
+
+  input: # any items you add here will show up as parameters
+    - species: "versicolor,virginica,setosa" # specify default values
+    
+  tasks:
+    export_to_google:
+      action: civis.import
+      with-items: 'class in <% $.species.split(",") %>' 
+      input:
+        name: 'Export Irises [<% $.class %>]'
+        sync_type: GdocExport
+        is_outbound: true
+        destination:
+          remote_host_id: 219
```

Comment:
Replace these with placeholders since whoever uses this will probably need different ones?

Reply:
done!